### PR TITLE
solution

### PR DESCRIPTION
**Pull request by yelnikov-andrii**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -3,16 +3,29 @@
...
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfValues = sourceString.split(';');
+  const obj = {};
+
+  for (let i = 0; i < arrayOfValues.length; i++) {
```
Comment (alexandra-protyanova) [inline src/convertToObject.js:20]: you can use forEach instead of for loop

**Review by alexandra-protyanova — APPROVED** [review]
👍

File: src/convertToObject.js [hunk 2]
```diff
@@ -3,16 +3,27 @@
...
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfValues = sourceString.split(';').map(item => item.split(':'));
```
Comment (Anna-Tkachenko) [inline src/convertToObject.js:17]:
```suggestion
  const arrayOfValues = sourceString
    .split(';')
    .map(item => item.split(':'));
```

**Review by Anna-Tkachenko — APPROVED** [review]
⭐️

**Pull request closed without merging**